src/Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights/ApplicationInsightsOutput.cs

```diff
@@ -6,17 +6,20 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+
```

Comment:
In this file the usings are grouped and sorted within the group.  The groups are system namespaces, non-system library namespaces and EventFlow namespaces.